src/graphql/node.rs

```diff
@@ -461,23 +462,23 @@ impl NodeStatus {
     async fn id(&self) -> ID {
         ID(self.id.to_string())
     }
-
+    /// The RAM size in bytes. The type must be u64.
```

Comment:
@sophie-cluml @div-seungha 
Users of GraphQL APIs will read this comment via the playground. It means this is supposed to explain what kind of value(s) the API returns. Therefore, I suggest:

```
/// The RAM size in bytes within the range representable by a `u64`
```

Reply:
I wasn't sure if the comment contained a valid or right content. I proceeded with the modification as you suggested and I think your suggestion is much useful for users to understand the meaning clearly.